recipes/mailio/all/conanfile.py

```diff
@@ -58,7 +58,7 @@ def layout(self):
 
     def requirements(self):
         self.requires("boost/1.81.0", transitive_headers=True)
-        self.requires("openssl/[>=1.1 <4]")
+        self.requires("openssl/[>=1.1 <4]", transitive_headers=True)
```

Comment:
Seems like `transitive_libs=True` for openssl will be needed :)
```suggestion
        self.requires("openssl/[>=1.1 <4]", transitive_headers=True, transitive_libs=True)
```

Reply:
Thank you for your help!